core/tests/puppeteer-acceptance-tests/utilities/common/test-constants.ts

```diff
@@ -140,7 +140,7 @@ export default {
   } as const,
 
   ViewportWidthBreakpoints: {
-    MOBILE_PX: 768,
+    MOBILE_PX: 769,
```

Comment:
I am not sure, why do you really need to change the viewports for the acceptance side. Please take a look again.

Reply:
This change is now removed.